deltachat-ios/DC/Wrapper.swift

```diff
@@ -157,6 +158,28 @@ class DcContext {
     func saveChatAvatarImage(chatId: Int, path: String) {
         dc_set_chat_profile_image(contextPointer, UInt32(chatId), path)
     }
+
+    @discardableResult
+    func addDeviceMessage(label: String, msg: DcMsg) -> Int {
+        return Int(dc_add_device_msg(contextPointer, label.cString(using: .utf8), msg.cptr))
+    }
+
+    func addWelcomeDeviceMessages() {
+        // create the device-chat by adding a message to it
+        let msg1 = DcMsg(type: DC_MSG_TEXT)
+        msg1.text = "Welcome to Delta Chat! – Delta Chat looks and feels like other popular messenger apps, "
```

Comment:
Why are you hardcoding the string here?

Reply:
it's just now. once the text is settled, it will go to transifex as usual.